### DISCUSSION TITLE
How do we use the extension ?

### DISCUSSION THREAD

**Discussion by lemmings189** [simplifylabs/remark#73] [Q&A]
Hello, on some website (ex. homepage of ytb) I can open the Remark's tab, but on others I cannot... (ex. this video https://www.youtube.com/watch?v=IWui73qXvfk)

Does anyone has a solution ?
Thanks

**Comment by simonkoeck** [comment]
Hey,

you can force-open the extension by clicking the remark icon in the extension bar. From there you can select when the widget is shown on a website. I think the default option is "Smart" which will only show the widget on the bottom right, if there's an existing remark on the website.

I hope I could help you :)

**Reply by lemmings189** [reply]
Hey,
Thanks a lot ! I just needed to reload my websites.
